src/App.tsx

```diff
@@ -20,12 +35,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(e) => {
+                  setQuery(e.target.value);
+                }}
```

Comment:
Create separate function for this event inside your `App` component.
This is a good practice and makes the code easy to maintain.


Checklist p1. Don't use one-letter variable naming
`[CODE STYLE] - Don't use one-letter variable naming. (It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search)`
```suggestion
                onChange={handleChange}
```